plugins/catalog-import/src/components/ImportComponentPage.tsx

```diff
@@ -57,14 +57,41 @@ export const ImportComponentPage = ({
     <Page themeId="home">
       <Header title="Register an existing component" />
       <Content>
-        <ContentHeader title="Start tracking your component on backstage">
+        <ContentHeader title="Start tracking your component in Backstage">
           <SupportButton>
-            Start tracking your component in Backstage. TODO: Add more
-            information about what this is.
+            Start tracking your component in Backstage by adding it to the
+            software catalog.
           </SupportButton>
         </ContentHeader>
-        <Grid container spacing={3} direction="column">
-          <Grid item>
+        <Grid container spacing={3} direction="row-reverse">
+          <Grid item xs={6}>
```

Comment:
@adamdmharvey If you want to tweak at a later point:

xs is just one of the screen sizes; there's md, lg, xl etc. You can specify multiple for different ranges. You may want to experiment so that

- both left and right are 6 at lg
- it's a 8 + 4 division at md
- it's a 12 + 12 division at xs (which also covers sm) which means it'll be two columns

or something like that!